dataactvalidator/config/sqlrules/fabs44_detached_award_financial_assistance_3.sql

```diff
@@ -8,9 +10,10 @@ FROM detached_award_financial_assistance AS dafa
 WHERE submission_id = {0}
     AND COALESCE(legal_entity_zip5, '') <> ''
     AND COALESCE(legal_entity_congressional, '') <> ''
-    AND NOT EXISTS (
-        SELECT 1
-        FROM zips AS z
-        WHERE z.zip5 = dafa.legal_entity_zip5
-            AND z.congressional_district_no = dafa.legal_entity_congressional
-    );
+    AND dafa.legal_entity_congressional
+        NOT IN (SELECT sc.congressional_district_no
```

Comment:
We don't want to do a `NOT IN` because that can REALLY slow things down when there are a lot of results in this SELECT (which may happen quite often). We want to stick to the NOT EXISTS method of doing this. It's not noticeable in a small submission or locally, but with a large submission and full zips table it can really take a performance hit

Reply:
Updating to use NOT EXISTS.